python/cudf/cudf/tests/indexes/index/test_properties.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+import pandas as pd
+import pytest
+
+import cudf
+from cudf.testing import assert_eq
+
+
+class TestIndexConstructorProperty:
```

Comment:
For all these tests:

1. For the index subclasses (e.g. `IntervalIndex`), can you put these tests in that subclass specific directory (`indexes/intervalindex`)
2. Can you put these tests in `test_attributes.py` (or create this file if it doesn't exists)?

Reply:
Done.